savant/deepstream/meta/frame.py

```diff
@@ -21,6 +24,7 @@ def __init__(
         super().__init__()
         self.batch_meta = frame_meta.base_meta.batch_meta
         self.frame_meta = frame_meta
+        self.savant_frame_meta = nvds_frame_meta_get_nvds_savant_frame_meta(frame_meta)
```

Comment:
The nvds_frame_meta_get_nvds_savant_frame_meta call will be executed for every frame, eg. in pyfunc. Are you sure it doesn't slow down the performance of the pipeline too much? Can we do it lazily?

Reply:
I moved the call of nvds_frame_meta_get_nvds_savant_frame_meta and now it will only be called once if we have not yet received GstFrameMeta